content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -38,23 +38,38 @@ https://www.youtube.com/watch?v=szzv4ZXmYAs
      [`mlem init`](https://mlem.ai/doc/command-reference/init), Iterative Studio
      will be able to parse the `.mlem` file to extract model metadata.
```

Comment:
I have some questions on the previous (existing) section. Sorry, I know it's not about these changes but it's closely related:

> If the model file is in a remote storage and is not DVC-tracked, enter the
     absolute path of the model file

What absolute path? Do we mean the remote's URL + path? Just the path after the remote's URL? An actual file system path to remote storage like `/mnt/ebs/username/something/dvcstore/a1/1f12fa2...`? Could be clearer. Maybe add examples.

Reply:
> If the model file is in a remote storage but is DVC-tracked ...
> If you use [MLEM] to save your model

What if you use MLEM + DVC? Which one applies?

Reply:
> use the path to the binary file that MLEM generates

What if you use [MLEM + DVC](https://mlem.ai/doc/use-cases/dvc)? Which one applies?

Also, what binary that MLEM generates? Are we talking about built model paks? Or whatever model is in the project (those are not generated by MLEM but rather by the ML code itself).

Reply:
> After you have run mlem init, Iterative Studio will be able to parse the .mlem file to extract model metadata

So use the .mlem file, or the binary file? This part is a bit confusing...

Reply:
Feel free to take over https://github.com/iterative/dvc.org/pull/4070 and address these questions there!

Reply:
Agreed, these need more work. Let's create a separate issue to address these.

Reply:
Have created https://github.com/iterative/dvc.org/issues/4075

Reply:
Thank you.